tommath_class.h

```diff
@@ -468,7 +464,6 @@
 #endif
 
 #if defined(BN_MP_INIT_MULTI_C)
-#   define BN_MP_ERR_C
```

Comment:
@sjaeckel There were many false positives in the dep.pl script like this one for example. Hopefully my fix removed all of them.

Reply:
The problem is also that comments are parsed. I should strip them from the source before applying the dep regex.

Reply:
looks very good!

Reply:
and fixed